tools/dds/dds-config/readme.md

```diff
@@ -0,0 +1,61 @@
+# dds-config tool
+
+## Goal
+This tool is used to configer DDS based device and ensure it operates correctly within a DDS environment.  
+It allows users to manage connection priorities, enabling the selection of either DDS or USB as the preferred connection method when both options are available.
```

Comment:
This is written in the description.
I think we can say here is allows the user to setup his device connection configuration